include/spirv/unified1/spirv.h

```diff
@@ -1223,6 +1224,7 @@ typedef enum SpvOp_ {
     SpvOpGroupSMaxNonUniformAMD = 5007,
     SpvOpFragmentMaskFetchAMD = 5011,
     SpvOpFragmentFetchAMD = 5012,
+    SpvOpReadClockKHR = 5056,
```

Comment:
Why doesn't this appear in the switch statement at the end of the file? Was this manually edited rather than generated?

Reply:
updated these files using the buildHeaders script.